openpype/plugins/publish/collect_anatomy_frame_range.py

```diff
@@ -0,0 +1,38 @@
+import pyblish.api
+
+
+class CollectAnatomyFrameRange(pyblish.api.InstancePlugin):
+    """Collect Frame Range specific Anatomy data.
+
+    Plugin is running for all instances on context even not active instances.
+    """
```

Comment:
This isn't true right? It doesn't run on inactive instances since it's currently an `InstancePlugin`?

```suggestion
    """Collect Frame Range specific Anatomy data."""
```

Reply:
This is started from stratch but anyways i have repharsed this docstring in the latest commit

Reply:
It's still incorrect. It doesn't run over inactive ones, right? And if so, I don't see a reason why we'd want to specify it either.

Note that there's currently a double space in front of **data** in the docstring.
Also, the plugin name and file still refer to it as "Anatomy data" so I'd refactor those too. (and thus with that the settings as well)